### DISCUSSION TITLE
What's the correct way to configure automatic translation and the score threshold?

### DISCUSSION THREAD

**Discussion by jredl-va** [WeblateOrg/weblate#10518] [Q&A] [answered]
We have configured weblate to use google translate and the weblate translation memory:

![image](https://github.com/WeblateOrg/weblate/assets/2300103/08a22e67-a3fa-4a1c-9539-6ebd1fb67926)

We set the score threshold to "100" thinking that this would only translate "exact matches" from the translation memory and the google translate would just fill in the blanks. 

However, with this configuration it seems as though nothing is being translated? I was hoping the community could help share the optimal configuration that you are using for this!

Thanks!

**Comment by nijel** [comment, marked as answer]
The threshold is in the current release also applied to services where it doesn't make much sense (it applies to machine translation hard-coded scoring). 

Since 63af6515c544074152310ba9f336426a839a6974 each service is responsible for applying it in a way that makes sense, so it will work as you have described.

If you want to achieve the same with the current release, you need two passes: first translation memory with 100 threshold and then Google with threshold 90 or lower.

**Reply by jredl-va** [reply]
Thank you!
